### PR TITLE
Swift 2.0.0 beta3

### PR DESCRIPTION
**Pull request by PeterMis**

### CODE REVIEW

File: lib/Entity/Module.php [hunk 1]
```diff
@@ -155,6 +155,10 @@ public function validate()
     {
         if (!v::intType()->validate($this->defaultDuration))
             throw new InvalidArgumentException(__('Default Duration is a required field.'), 'defaultDuration');
+
+        if (!v::alnum(',')->validate($this->validExtensions)) {
```
Comment (dasgarner) [inline lib/Entity/Module.php]: Does this work if its empty?

File: lib/Factory/DisplayFactory.php [hunk 2]
```diff
@@ -120,6 +120,7 @@ public function getById($displayId, $showTags = false)
     public function getByLicence($licence)
     {
         $displays = $this->query(null, ['disableUserCheck' => 1, 'license' => $licence]);
+        $this->getLog()->debug('DISPLAY LOG LICENCE ' . $licence );
```
Comment (dasgarner) [inline lib/Factory/DisplayFactory.php]: if we need to leave this in, can it be reworded and not capitals?

Comment (PeterMis) [reply lib/Factory/DisplayFactory.php]: we don't need it, there is a chance I forgot to remove it.

File: lib/Entity/Module.php (hunk 1 above)
Comment (PeterMis) [reply lib/Entity/Module.php]: no, it does not...

Do we want something like that? 🤔 

```
        if (v::notEmpty()->validate($this->validExtensions)) {
            if (!v::alnum(',')->validate($this->validExtensions)) {
                throw new InvalidArgumentException(__('Comma separated file extensions only please, without the .'),'validExtensions');
            }
        }
```

Comment (dasgarner) [reply lib/Entity/Module.php]:
```
 if (!empty($this->validExtensions) && !v::alnum(',')->validate($this->validExtensions)) {
            throw new InvalidArgumentException(__('Comma separated file extensions only please, without the .'), 'validExtensions');
        }
```

Did you mean that?

**Pull request merged**